src/prototype_room_creepbuilder.js

```diff
@@ -117,23 +117,49 @@ Room.prototype.spawnCheckForCreate = function(creepsConfig) {
 
     let creep = this.memory.queue[0];
     energyNeeded = 50;
-
-    if (this.spawnCreateCreep(creep.role, creep.heal, creep.level, creep.squad, creep.routing, creep.base)) {
+    let c_room = this;    
```

Comment:
Removed this line as it is unneeded. This is aliased to room at the top of the if block.  Also modifying the rest of the file to follow the correct idiom